src/hooks/useShowcaseTokens.js

```diff
@@ -20,10 +21,16 @@ export default function useShowcaseTokens() {
   const addShowcaseToken = useCallback(
     async asset => {
       dispatch(rawAddShowcaseToken(asset));
-      dispatch(setOpenFamilyTabs({ index: 'Showcase', state: true }));
+      updateOpenFamilies({ 'Showcase-showcase': true });
```

Comment:
Do we definitely want 'Showcase-showcase' instead of just `Showcase` here? (based off of the default value returned in `useOpenFamilies`)

Reply:
yes, my mistake